### DISCUSSION TITLE
深入理解K8s资源限制

### DISCUSSION THREAD

**Discussion by giscus** [qingwave/qingwave.github.io#84] [Announcements]
# 深入理解K8s资源限制

https://qingwave.github.io/understanding-resource-limits-in-kubernetes/

**Comment by qingwave** [comment]
本文关于Cgroup的目录都是基于CgroupV1操作的，目前新版K8s已经已经支持Cgroupv2，路径已经与文章有差异，需要注意下。

但具体原理还是一样的，另外对于非docker的Container Runtime可以通过[critool](https://kubernetes.io/zh-cn/docs/tasks/debug/debug-cluster/crictl/)来查看容器的相关配置
